doc/CHANGES.md

```diff
@@ -2,6 +2,7 @@
 
 ## Highlighted Features
 
+* Add example logrotate configuration (Graeme Porteous)
```

Comment:
be good to add an upgrade note of how to generate it if people haven't already created their own